bin/constants.js

```diff
@@ -0,0 +1,37 @@
+const NON_COMPILATION_ARGS = [
+	"init",
+	"migrate",
+	"add",
+	"remove",
+	"serve",
+	"generate-loader",
+	"generate-plugin",
+	"info"
+];
+
+const CONFIG_GROUP = "Config options:";
+const BASIC_GROUP = "Basic options:";
+const MODULE_GROUP = "Module options:";
+const OUTPUT_GROUP = "Output options:";
+const ADVANCED_GROUP = "Advanced options:";
+const RESOLVE_GROUP = "Resolving options:";
+const OPTIMIZE_GROUP = "Optimizing options:";
+const DISPLAY_GROUP = "Stats options:";
+const GROUPS = {
+	CONFIG_GROUP,
+	BASIC_GROUP,
+	MODULE_GROUP,
+	OUTPUT_GROUP,
+	ADVANCED_GROUP,
+	RESOLVE_GROUP,
+	OPTIMIZE_GROUP,
+	DISPLAY_GROUP
+};
+
+const webpackOptionsFlag = "WEBPACK_OPTIONS";
```

Comment:
This should be all capital case, as it's a constant like the others

Reply:
Makes sense, I initially just copied whatever was there, updated 👍